pydy/tests/test_models.py

```diff
@@ -19,8 +16,8 @@ def test_multi_mass_spring_damper():
 
     sys = multi_mass_spring_damper()
 
-    assert Counter(sys.constants_symbols) == Counter([k0, c0, m0])
-    assert sys.specifieds_symbols == list()
+    assert sys.constants_symbols == {k0, c0, m0}
```

Comment:
Does this now return a set? If so, I don't think we should change this, as all the code in the wild depends on this being an ordered sequence.